### PR TITLE
Manage personnel ToolBar and PopOverMenu

### PR DESCRIPTION
**Pull request by olerichard**
Added a toolbar componenet, handling the buttons used in the managed personnel page. 
Added a popover menu in the sidesheet. 
The menu for now only holds a delete button. 
The button is only active for lines thats been created during that "edit session". 
You can not delete already created users from the edit view. 

the popovermenu could need some scruteny.
the menu only closes if you click a button or the dots again. 
Have tried some different solution similar to how useToolTipRef is setup, but didnt really work for me. 
Will need som help on that part, and solve that in another PR later on. 

There has been added a new verb to the personnelReducer. 
that will remove entries supplied. to be used when deleting users for example.

### CODE REVIEW

File: src/frontend/src/pages/ProjectPage/pages/ContractPage/pages/ManagePersonnelPage/components/ManagePersonnelToolBar/index.tsx [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+};
+
+export type ToolBarProps = {
+    addButton?: IconButtonProps,
+    deleteButton?: IconButtonProps,
+    editButton?: IconButtonProps,
+};
+
+const ManagePersonnelToolBar: React.FC<ToolBarProps> = ({ addButton, deleteButton, editButton }) => {
+    return <div className={styles.container}>
```
Comment (eskil0312) [inline src/frontend/src/pages/ProjectPage/pages/ContractPage/pages/ManagePersonnelPage/components/ManagePersonnelToolBar/index.tsx]: Is this formatted?

Comment (olerichard) [reply src/frontend/src/pages/ProjectPage/pages/ContractPage/pages/ManagePersonnelPage/components/ManagePersonnelToolBar/index.tsx]: what do you mean ?

Comment (eskil0312) [reply src/frontend/src/pages/ProjectPage/pages/ContractPage/pages/ManagePersonnelPage/components/ManagePersonnelToolBar/index.tsx]: Formatted with Prettier

Comment (olerichard) [reply src/frontend/src/pages/ProjectPage/pages/ContractPage/pages/ManagePersonnelPage/components/ManagePersonnelToolBar/index.tsx]: no, the extension was not installed in the "container". 
Didnt know that was a thing.. 
strange

File: src/frontend/src/pages/ProjectPage/pages/ContractPage/pages/ManagePersonnelPage/components/PopOverMenu/index.tsx [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+import Arrow from './Arrow';
+import classNames from 'classnames';
+
+export type TooltipPlacement = 'below' | 'above' | 'left' | 'right';
+
+type PopOverMenuProps = {
+    label?: String;
+    placement?: TooltipPlacement;
+};
+const PopOverMenu: React.FC<PopOverMenuProps> = ({ label, placement = 'below', children }) => {
```
Comment (MartinForre) [inline src/frontend/src/pages/ProjectPage/pages/ContractPage/pages/ManagePersonnelPage/components/PopOverMenu/index.tsx]: Could we use the exising popover instead?

**Pull request merged**